### PR TITLE
build: 'make clean' removes parse-datetime.c

### PR DESCRIPTION
**Pull request by giuseppe**
and fix make dist while at it.

Signed-off-by: Giuseppe Scrivano <gscrivan@redhat.com>

### CODE REVIEW

File: Makefile-libostree.am [hunk 1]
```diff
@@ -182,7 +182,7 @@ pkgconfig_DATA += src/libostree/ostree-1.pc
 
 gpgreadme_DATA = src/libostree/README-gpg
 gpgreadmedir = $(pkgdatadir)/trusted.gpg.d
-EXTRA_DIST += src/libostree/README-gpg src/libostree/bupsplit.h
+EXTRA_DIST += src/libostree/README-gpg src/libostree/bupsplit.h src/libostree/ostree-enumtypes.h.template src/libostree/ostree-enumtypes.c.template src/libostree/ostree-deployment-private.h
```
Comment (mbarnes) [inline Makefile-libostree.am]: Minor, but maybe split this into multiple lines?

**Comment by mbarnes** [comment]
I know we don't use `make dist` for releases but I see no harm in keeping it working.  :+1:

**Comment by cgwalters** [comment]
Yeah I just sign git tags, but I don't mind changing `EXTRA_DIST` either.  Changing `CLEANFILES` makes sense for sure.  So agreed with @mbarnes :+1:

**Comment by giuseppe** [comment]
thanks, I've split the long line and pushed the patch.

**Pull request closed without merging**